spec/support/request/web_helper.rb

```diff
@@ -120,7 +120,7 @@ def select2_select(value, options)
   end
 
   def open_select2(selector)
-    page.find(selector).find(:css, '.select2-choice, .select2-search-field').click
+    page.find(selector).scroll_to(page.find(selector)).find(:css, '.select2-choice, .select2-search-field').click
```

Comment:
so the element wasn't present in the viewport or something?

Reply:
Yeah, that sounds like the issue. This may actually fix some of the other flaky specs? Not sure.